components/org.wso2.carbon.identity.data.publisher.authentication.audit/src/main/java/org/wso2/carbon/identity/data/publisher/authentication/audit/AuthenticationAuditLoggingHandler.java

```diff
@@ -123,7 +127,7 @@ protected void doPublishAuthenticationStepSuccess(AuthenticationAuditData authen
 
         AUDIT_LOG.info(String.format(
                 FrameworkConstants.AUDIT_MESSAGE,
-                authenticationData.getAuthenticatedUser(),
+                maskedUsername,
```

Comment:
is the user id always not available in this case?

Reply:
Yes, it is not available in this scenario.